components/Nav.js

```diff
@@ -18,12 +16,12 @@ const links = {
   Leaderboard: {
     href: '/leaderboard',
   },
+  'Stories, Art, & Toons:' {
```

Comment:
Just noticed this as well. the colon needs to be after the closing quote mark for this to be valid code.